Update Leadership Resources

briaugenreich opened the pull request:
I tired to put all of our resources for new leaders and existing leaders into this repo. As well as our chapter orginization. 

@trishmahan @michelematthews

briaugenreich committed:
Update chapter organization readme

1. Update quick links
2. Rearrange details
3. Add descriptions to RVA resources

briaugenreich committed:
Update chapter orgnization readme

1. Rearrange order
2. Add description to RVA resources
3. Update links

briaugenreich committed:
Remove channels guide from slack guide

briaugenreich committed:
Update Code of Conduct

1. added the content from the global organization's code of
conduct

briaugenreich committed:
Update global github link

briaugenreich committed:
Updating leader resources with our leaders

1. Updated leader resources read me with new table of contents, intro,
and team descriptions
2. Updated our leaders with current leads and new team structure

OUTSTANDING -> all leads update their name with a link to their social
site or personal site of choice. (submit PRs)

briaugenreich committed:
Add roles to network team (our leaders)

briaugenreich committed:
Update leader resources

Began making all the leader resources complete and robust.

1. Add becoming a leader details
2. event play book and ideas (needs refinement)
3. Leadership overview contains expectations and responsibilities for
most roles (needs refinement)
4. Added a goals section for us to track what we are aiming for (needs
refinement)

briaugenreich committed:
Add github responsibility to leadership overview

michelematthews commented on leadership-resources/our_leaders.md:
```diff
@@ -1,53 +1,50 @@
+- Director: [Elisa Parkins](https://twitter.com/elisa_parkin?lang=en) & [Michele Matthews](https://twitter.com/mlmatthews?lang=en)
 - Content Lead:
-  - OPEN
+  - OPEN 
```
Add Siennia (Sie Sie) Coleman

ghost commented:
I left one in-line comment to add Sie Sie -- otherwise, this looks awesome.  Thanks so much for adding this.

briaugenreich committed:
Format community repos link in README

briaugenreich committed:
Update link to short code of conduct on readme

briaugenreich committed:
Update README.md

1. add contributing details
2. add credits section

briaugenreich committed:
Remove outdated contributing.md 

Removed the content that i do not believe is relevant. Need to update

briaugenreich committed:
Rename code_of_conduct.md to code_of_conduct_FULL.md

Hotlink to short code of conduct is in the readme. This is the full version for anyone who is interested.

briaugenreich committed:
Update README.md

include in the resources description for leadership resources the team roster

briaugenreich committed:
Update slack signup link

Made the slack signup link a clickable hotlink

briaugenreich committed:
Update meetup link

briaugenreich committed:
Update README.md

briaugenreich committed:
Formatting and content changes for new leaders page

briaugenreich committed:
Format leadership exepcations

briaugenreich committed:
update welcome for readme

briaugenreich committed:
Update README.md

briaugenreich committed:
Update README.md

briaugenreich committed:
Update README.md

briaugenreich committed:
Update our_leaders.md

briaugenreich committed:
Update event-inspiration.md

briaugenreich committed:
Update event-playbook.md

briaugenreich committed:
Update python leads

briaugenreich committed:
Update our event playbook sepcifics 

Copied sample from dc chapter orginization and filled in our coordination details 

TODO fill in links to trello board and sample decks.

briaugenreich committed:
Update event-playbook.md

briaugenreich committed:
Update network-goals.md

briaugenreich committed:
Update network-goals.md

briaugenreich committed:
Complete roles and responsibilities

briaugenreich committed:
Update event-playbook.md

Commit:
Add powerpoint samples

Commit:
Update to Slack Guide

Commit:
Update leadership resources readme

Commit:
Update becoming_a_leader.md

Commit:
Update becoming a leader

Commit:
Update leadership overview

briaugenreich committed:
Update leadership_overview.md

briaugenreich committed:
Merge branch 'master' into bri-changes
